Feature: Implemented interface for LeaseKeepAlive RPC

timstudd opened the pull request:
* Implemented interface for LeaseKeepAlive RPC
* Performed general cleanup
* Incremented package version

timstudd committed:
Implemented interface for LeaseKeepAlive RPC

General cleanup

Version bump

Added test

Remove emit

Add emit and error handling

ychen11 commented:
@timstudd Thanks for contributing to this repo! 
3 things:
1. Please add related test cases, etcd api always works I know, but tests would help us to avoid making mistakes.
2. Please add the changed proto file currently, I will extract the proto file as a sub repo directly linked to etcd later. But now please keep this like before :)
3. Originally, I did not think this repo would be forked. However, now you forked it, I personally think *squash* will be better for merging

timstudd commented:
@ychen11 I added the necessary test and squashed the commits.  Concerning 2, there were no changes to the proto file - the proto file in your repo already had the LeaseKeepAlive RPC. 

Please let me know if you desire any other changes.
